crates/flowctl/src/raw/spec.rs

```diff
@@ -1,17 +1,17 @@
 use crate::local_specs;
 use anyhow::Context;
-use proto_flow::{capture, flow};
+use proto_flow::{capture, flow, materialize};
 
 #[derive(Debug, clap::Args)]
 #[clap(rename_all = "kebab-case")]
 pub struct Spec {
     /// Path or URL to a Flow specification file.
     #[clap(long)]
     source: String,
-    /// Name of the capture to discover within the Flow specification file.
-    /// Capture is required if there are multiple captures in --source specifications.
+    /// Name of the task (capture or materialization) to request the spec for.
```

Comment:
I kinda feel like this ought to support derivations, too, if that wouldn't be too difficult. Or can defer until we actually need it, but it might be worth noting that explicitly in the comment if we're not going to support them yet.

Reply:
Hmm yeah I didn't even realize that derivations had a spec request / response, but I guess it's in the protocol. I'll work on adding that when I get back.

Reply:
Also note to self: `flowctl preview` uses the flag `--name` to specify which capture/derivation/materialization should be run, so that it would probably be best to be consistent with that.